File: src/main/java/de/hpi/tdgt/test/time_measurement/MqttTimeMessage.java
package de.hpi.tdgt.test.time_measurement;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Map;
import java.util.concurrent.ConcurrentHashMap;


@NoArgsConstructor
@Getter
@Setter
public class MqttTimeMessage {
    private Map<String, Map<String, Map<String, Map<String, String>>>> times = new ConcurrentHashMap<>();
    private long testId;
    private long creationTime;
}

File: src/main/java/de/hpi/tdgt/test/time_measurement/MqttTimeMessage.kt
package de.hpi.tdgt.test.time_measurement

import java.util.concurrent.ConcurrentHashMap

class MqttTimeMessage {
    var times: Map<String, Map<String, Map<String, Map<String, String>>>> = ConcurrentHashMap()
    var testId: Long = 0
    var creationTime: Long = 0
    var node: String = "unknown node"
}